health-services/project-factory/src/server/utils/excelUtils.ts

```
@@ -231,7 +432,7 @@ function finalizeSheet(request: any, sheet: any, frozeCells: boolean, frozeWhole
   let columnsToBeFreezed: any[] = [];
   let columnsToHide: any[] = [];
   if (fileUrl) {
-    columnsToHide = ["HCM_ADMIN_CONSOLE_BOUNDARY_CODE_OLD",...schema?.columnsToHide];
+    columnsToHide = ["HCM_ADMIN_CONSOLE_BOUNDARY_CODE_OLD", ...schema?.columnsToHide];
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)